### ISSUE TITLE
Garbled character when launching tmux

### ISSUE THREAD

**Issue by yfzm** [microsoft/terminal#6950]
# Environment

```none
Windows build number: 10.0.18363.0
Windows Terminal version (if applicable): 1.0.1811.0
tmux version: 2.1
SSH target machine: 4.15.0-46-generic # 49~16.04.1-Ubuntu SMP x86_64 GNU/Linux
```

# Steps to reproduce

1. ssh to a Linux host
2. open a tmux session

# Expected behavior
Characters should be rendered normally.

# Actual behavior
There are some garbled characters, as shown in this picture:
![image](https://user-images.githubusercontent.com/21277160/87699375-58c5d180-c7c7-11ea-8aea-1807193400f3.png)

I tried to copy the garbled characters to clipboard, and displayed in hex editor. It showed that the ascii code of abnormal character is `0x0f`.

BTW, this problem only exists in **this** PC. I don't know if I have some improper configuration with this PC. And the garbled characters disappear if I run tmux in the native terminal in Linux. Also, I tried to use Mobaxterm, which is also normal.

**Comment by DHowett** [comment]
I believe this was fixed in 1.1; can you try the Preview channel?

FWIW: #3487, original report at #176 which had similar symptoms ("only happens on one computer, we can't reproduce it")

**Comment by DHowett** [comment]
Actually, because this is 0x0f (thank you !!!) it is definitely a /duplicate of #3487
